packages/contracts-sdk/src/interfaces/objects/AuthenticatorData.ts

```diff
@@ -0,0 +1,10 @@
+import { JSONString } from "@snickerdoodlelabs/objects";
+import { BytesLike } from "ethers";
+
+export class AuthenticatorData {
```

Comment:
Need a method in CryptoUtils parseClientDataJson(clientDataJSON: JSONString): Result<AuthenticatorData, whatever>